modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.scss

```diff
@@ -0,0 +1,11 @@
+.lfr__objects-custom-vertical-bar--left-closed.c-slideout-start,
+.lfr__objects-custom-vertical-bar--right-closed.c-slideout-end {
+	top: 129px;
+}
+
+.lfr-objects__model-builder-content {
+	display: flex;
+	height: calc(100vh - 129px);
+	overflow-x: hidden;
+	width: 100%;
+}
```

Comment:
When we press the eye button to close the sidebars, I can see the sidebars go down a little before closing:

![screenshot-1697569450](https://github.com/liferay-objects/liferay-portal/assets/61854510/62587024-128d-4e32-9810-2a224d790e32)


To resolve this small behavior, we can replace the uses of 129px with 121 px.

```
.lfr__objects-custom-vertical-bar--left-closed.c-slideout-start,
.lfr__objects-custom-vertical-bar--right-closed.c-slideout-end {
	top: 129px;
}

.lfr-objects__model-builder-content {
	display: flex;
	height: calc(100vh - 129px);
	overflow-x: hidden;
	width: 100%;
}
```

Reply:
@igor-franca Thanks for noticing that! I'll apply the fix :)